src/api/app/models/bs_request.rb

```diff
@@ -157,6 +157,8 @@ def self.new_from_xml(xml)
     else
       theid = nil
     end
+    # we will set it our own according to the user
+    hashed.delete('creator')
```

Comment:
On Dienstag, 14. Juni 2016, 01:33:02 CEST wrote Bj�rn Geuken:
> > @@ -157,6 +157,8 @@ def self.new_from_xml(xml)
> >      else
> >        theid = nil
> >      end
> > +    # we will set it our own according to the user
> > +    hashed.delete('creator')
> 
> What is this needed for?

some lines below is a check if every xml content is handled
or it will barf.

Since we don't want to let the user setting this we need to remove
it from xml hash to avoid the error.

We could think about to let admin users setting it though ...

-- 

Adrian Schroeter
email: adrian@suse.de

SUSE Linux GmbH, GF: Felix Imend�rffer, Jane Smithard, Graham Norton, HRB 21284 (AG N�rnberg) 
 
Maxfeldstra�e 5                         
90409 N�rnberg 
Germany